.github/workflows/build_orca.yml

```diff
@@ -194,7 +194,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: OrcaSlicer_Windows_${{ env.ver }}_portable
-          path: ${{ github.workspace }}/build/OrcaSlicer_Windows_${{ env.ver }}_portable.zip
+          path: ${{ github.workspace }}/build/OrcaSlicer
```

Comment:
IMO this is the only change needed to solve the problem.

Reply:
If you say so, it should work and I'm happy to follow. 😊